models/__init__.py

```diff
@@ -4,20 +4,20 @@
 from time import gmtime, strftime
 
 def _get_all_piskels_for_user(user_id, limit=20):
-  mem_key = "user_piskels_" + str(user_id)
+  mem_key = 'user_piskels_' + str(user_id)

```

Comment:
It wasn't consistent throughout the code.
For example https://github.com/juliandescottes/piskel-website/blob/master/handlers/piskel.py

So I changed them following this rule: If there is no single quote in the text then single quotes, otherwise double quotes.
Though it's not that important so I can remove the commit if you think this is unnecessary.